internal/npm_install/generate_build_file.ts

```diff
@@ -477,7 +509,17 @@ def _maybe(repo_rule, name, **kwargs):
  * Generate build files for a scope.
  */
 async function generateScopeBuildFiles(scope: string, pkgs: Dep[]) {
-  const buildFile = generateBuildFileHeader() + printScope(scope, pkgs);
+  pkgs = pkgs.filter(pkg => !pkg._isNested && pkg._dir.startsWith(`${scope}/`));
+  let deps: Dep[] = [];
+  pkgs.forEach(pkg => {
+    deps = deps.concat(pkg._dependencies.filter(dep => !dep._isNested && !pkgs.includes(pkg)));
+  });
+  // filter out duplicate deps
+  deps = [...pkgs, ...new Set(deps)];
+
+  let buildFile = config.exports_directories_only ?
+      printScopeExportsDirectories(scope, deps) :
+      printScope(scope, deps);
```

Comment:
```    
if (isSymbolicLink && config.exports_directories_only && path.basename(path.dirname(fullPath)) == '.bin') {
      // Delete any .bin symlinks if exports_directories_only is on since these can point to outside of the
      // tree artifact on linux which breaks that package's copy_directory.
      await fs.unlink(fullPath);
      continue
    }
```